test/Integration/SqlOutputBindingJSIntegrationTests.cs

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Tests.Integration
+{
+    [Collection("IntegrationTests")]
+    public class SqlOutputBindingJSIntegrationTests : IntegrationTestBase
+    {
+        private readonly string workingDirectoryFolder = "samples-js";
+        public SqlOutputBindingJSIntegrationTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private async Task<HttpResponseMessage> SendOutputRequest(string functionName, IDictionary<string, string> query = null)
```

Comment:
We shouldn't be needing to duplicate all this code - the tests to verify the APIs should be the same across all the languages. The only thing that we should expect to change is how the function host is started up.

Reply:
All I did is create a separate class and duplicated this one SendRequest method for easier management, if we add them all in once class, I have to name the tests tagging with language to make that distinction. I thought this was cleaner.
In this case it's justified because the c# output binding tests use Get requests for updating data as well (since they use query strings for reading the product data) But javascript samples I added, am reading the product data from body. So I have to call post request.